src/interpolation.js

```diff
@@ -93,7 +93,7 @@ exports.colorRGBToHexa = function(t, a, b) {
 	return '#' + cr.toString(16) + cg.toString(16) + cb.toString(16);

```

Comment:
I think this breaks on low numbers, no? `3..toString(16)` will return `"3"`, not `"03"`.